wasmtime/_ffi.py

```diff
@@ -53,6 +53,7 @@
 WASMTIME_EXTERN_GLOBAL = c_uint8(1)
 WASMTIME_EXTERN_TABLE = c_uint8(2)
 WASMTIME_EXTERN_MEMORY = c_uint8(3)
+WASMTIME_EXTERN_SHAREDMEMORY = c_uint8(4)
```

Comment:
This was wierd, because the `4` is shared with the definition below (`WASMTIME_EXTERN_INSTANCE = c_uint8(4)`) but `4` is also how wasmtime runtime defines this const.

Reply:
Good catch! The `WASMTIME_EXTERN_{INSTANCE,MODULE}` types should actually be removed since the C API hasn't had support for those in quite awhile.